spec/index.html

```diff
@@ -240,7 +240,27 @@ <h2>Uses of Dataset Canonicalization</h2>
       RDF store or serialization is arbitrary,
       and typically not relatable to the context within which it is used.</p>
 
-    <p>This specification defines an algorithm for creating stable <a>blank node identifiers</a> repeatably for different serializations possibly using individualized <a>blank node identifiers</a> of the same RDF graph (dataset) by grounding each <a>blank node</a> through the nodes to which it is connected, essentially creating <em>Skolem <a>blank node identifiers</a></em>. As a result, a graph signature can be obtained by hashing a canonical serialization of the resulting <a>normalized dataset</a>, allowing for the isomorphism and digital signing use cases. As blank node identifiers can be stable even with other changes to a graph (dataset), in some cases it is possible to compute the difference between two graphs (datasets), for example if changes are made only to ground triples, or if new blank nodes are introduced which do not create an automorphic confusion with other existing blank nodes. If any information which would change the generated blank node identifier, a resulting diff might indicate a greater set of changes than actually exists.</p>
+    <p>This specification defines an algorithm for creating stable
+      <a>blank node identifiers</a> repeatably for different serializations
+      possibly using individualized <a>blank node identifiers</a>
+      of the same RDF graph (dataset) by grounding each <a>blank node</a>
+      through the nodes to which it is connected,
+      essentially creating <em>Skolem <a>blank node identifiers</a></em>.
+      As a result, a graph signature can be obtained by hashing a canonical serialization
+      of the resulting <a>normalized dataset</a>,
+      allowing for the isomorphism and digital signing use cases.
+      As blank node identifiers can be stable even with other changes to a graph (dataset),
+      in some cases it is possible to compute the difference between two graphs (datasets),
+      for example if changes are made only to ground triples,
+      or if new blank nodes are introduced which do not create an automorphic confusion
+      with other existing blank nodes.
+      If any information which would change the generated blank node identifier,
+      a resulting diff might indicate a greater set of changes than actually exists.
+      Additionally, if the starting dataset is an N-Quads document,
+      which may be based on a previously canonicalized dataset,
+      it may be possible to correlate the original blank node identifiers
```

Comment:
```suggestion
      Additionally, if the starting dataset is an N-Quads document,
      it may be possible to correlate the original blank node identifiers
```

The fact whether the N-Quads comes from a previously canonicalized dataset or not is irrelevant imho.